crates/ruff_linter/src/rules/flake8_pie/rules/non_unique_enums.rs

```diff
@@ -103,3 +93,41 @@ pub(crate) fn non_unique_enums(checker: &mut Checker, parent: &Stmt, body: &[Stm
         }
     }
 }
+
+fn member_has_specialized_value(checker: &Checker, expr: &Expr) -> bool {
```

Comment:
I'd follow the terminology used in the [mypy blog](https://mypy-lang.blogspot.com/2024/12/mypy-114-released.html): `is_enum_member_with_unknown_value_in_stub`. That means I'd also move the `enum.auto` handling out of this function, but we can then inline `is_casted_ellipsis`